DFE-Digital/sts-plan-technology-for-your-school
# feat: replace recommendation page breadcrumbs with back button

## Overview

Addresses ticket [234122](https://dfe-ssp.visualstudio.com/s190-schools-technology-services/_boards/board/t/Team%20Build/Stories/?workitem=234122)

## Changes

- Remove breadcrumbs from top of recommendation page
- Replace with back button

## How to review the PR

- Navigate to a  recommendation page
- Verify breadcrumbs are no longer there
- Verify back button exists
- Verify back button takes you back to the self assessment page

## Checklist

Delete any rows that do not apply to the PR.
